src/Payment.php

```
@@ -136,6 +136,19 @@ public function setCardOperationType(string $type): Payment
         return $this;
     }
 
+    /**
+     * Automatically converts given array into base64 and sets it to booking_info parameter
+     *
+     * @param array $info
+     * @return $this
+     */
+    public function setBookingInfo(array $info): Payment
+    {
+        $base64 = base64_encode(json_encode($info));
```

Hi! json_encode can return false, and in that case we should throw an exception.

---

Updated in newest commit.